src/neuroconv/tools/nwb_helpers.py

```diff
@@ -115,6 +118,58 @@ def add_device_from_metadata(nwbfile: NWBFile, modality: str = "Ecephys", metada
             nwbfile.create_device(**dict(defaults, **dev))
 
 
+def wrap_data_in_H5DataIO(data, maxshape=None, chunks=None, compression=None, compression_opts=None) -> H5DataIO:
+    """
+    Auxiliar function to wrapp data in a :py:class:`hdmf.backends.hdf5.h5_utils.H5DataIO` class. This is used in
+    neuroconv to have a centralized place to check for valid parameter combinations.
+
+    Parameters
+    ----------
+    data : np.darray, list or iterable, optional
+        The data to wrapp
+    maxshape : tuple, optional
+        Dataset will be resizable up to this shape , by default None
+    chunks : bool, tuple, optional
+        Chunk shape or True to enable auto-chunking, by default None
+    compression : str, bool, optional
+        Compression strategy. If a bool is given, then gzip compression will be used by default.
+        http://docs.h5py.org/en/latest/high/dataset.html#dataset-compression, by default None
+    compression_opts : _type_, optional
+        Parameter for compression filter, by default None
+
+    Returns
+    -------
+    H5DataIO
+        The wrapped data in an H5DataIO object.
+    """
+
+    valid_options_for_compression = [
+        "gzip",
+        "lzf",
+        None,
+        True,
+    ]
```

Comment:
Why are you limiting to these values?

Reply:
See https://github.com/NeurodataWithoutBorders/pynwb/pull/1529

Reply:
I took it from existing code and added None and True options.